package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "1.9.19",
+  "version": "2.1.1",
```

Comment:
> There is no binaries available for v2.1.0. So v2.1.1 is the only choice.

The readme snippet @jzaefferer mentioned says only major & minor needs to agree with PhantomJS so 2.1.0 would be fine.